packages/core/src/fonts.ts

```diff
@@ -29,8 +29,11 @@ const fonts = {
       ingress: sizes('20px', '31px'),
       button: sizes('16px', '24px'),
       content: sizes('18px', '29px'),
+      metaTextxxsmall: sizes('10px', '12px'),
```

Comment:
blir ikke det rart å blande både camelcase og ikke camelcase? jeg vett det ser litt rart ut men burde ikke vi heller kalle det  metaTextXxsmall for kontinuitet?

Reply:
Jeg drev å tenkte det samme, men usikker. Kan sikkert gjøre det! :D